cv/.gitignore

```diff
@@ -0,0 +1,51 @@
+devel/
```

Comment:
@spencerteetaert 

Was this change necessary? We can move it to the root if you think it is